inventories/sentry/inventory.yml

```diff
@@ -0,0 +1,14 @@
+---
+sentry:
+  hosts:
+    sentry.nos.social:
```

Comment:
We can simplify the inventory file, because of (now documented) changes to our certbot role.  If there is no domain var, we will just use the inventory_hostname, so we no longer need the domain list or wildcard_needed vars.